crates/runtime/benches/bench_search/mod.rs

```diff
@@ -54,7 +57,8 @@ impl Display for SearchBenchmarkResultBuilder {
           search_time: {:.2} ms,\n\
           rps: {:.2},\n\
           mean_response_time: {:.2} ms,\n\
-          p95_response_time: {:.2} ms\n",
```

Comment:
I think you can remove need escaped newlines via
```rust
        write!(
            f,
    r#"configuration: {},
  status: {},
  run_id: {},
  commit_sha: {},
  branch_name: {},
  started_at: {},
  index_time: {:.2} ms,
  search_time: {:.2} ms,
  rps: {:.2},
  mean_response_time: {:.2} ms,
  p95_response_time: {:.2} ms,
  precision: {:.2}"#
```

Reply:
Thank you 🙏 - updated